data_safe_haven/commands/deploy_shm_command.py

```diff
@@ -115,7 +121,7 @@ def handle(self) -> None:
     def add_missing_values(self, config: Config) -> None:
         """Request any missing config values and add them to the config"""
         # Request FQDN if not provided
-        fqdn = self.option("fqdn")
+        fqdn = cast(str, self.option("fqdn"))
```

Comment:
Does this just trick the type checker that the result of `self.option("fqdn")` is always a string?

What if it is empty?
What if it isn't a string?

Reply:
This is a limitation in the type signature of `Command.option`. It returns `dict[Unknown, Unknown] | Unknown | Any | Literal[False]` which are all of the possibilities that `cleo` can coerce the command-line arguments into.

However, this isn't very useful for type checking. I'll have another think about this.